### ISSUE TITLE
csp adapter compliance message typo

### ISSUE THREAD

**Issue by brudnak** [rancher/rancher#38024]
## Rancher Server Setup

| Component               | Version / Type                                                                     |
| ----------------------- | ---------------------------------------------------------------------------------- |
| Rancher version         | v2.6-8b3919b99293fc8b0dc5fe7191ca5bec9609aadd-head                                 |
| Rancher commit link     | https://github.com/rancher/rancher/commit/8b3919b99293fc8b0dc5fe7191ca5bec9609aadd |
| Installation option     | `eksctl` eks local cluster (bring your own cluster)                                |
| local k8s cluster       | v1.22.9-eks-a64ea69                                                                |
| Cert Details            | let's encrypt                                                                      |
| Helm version            | v2.16.8-rancher1                                                                   |
| Downstream cluster type | rke1 linode                                                                        |
| Downstream K8s version  | v1.23.6-rancher1-1                                                                 |
| Logged in user role     | administrator                                                                      |
| Browser type            | google chrome                                                                      |
| Browser version         | 102.0.5005.115 (Official Build) (x86_64)                                           |

**ADDITIONAL SETUP**

- eksctl install command:

```shell
eksctl create cluster \
  --name $YOUR_CLUSTER_NAME \
  --version 1.22 \
  --region us-east-2 \
  --nodegroup-name $YOUR_NODE_GROUP_NAME \
  --nodes 3 \
  --nodes-min 1 \
  --nodes-max 4 \
  --managed
```

- Helm commands for Rancher installation:
  
```shell
helm install rancher rancher-latest/rancher \
  --namespace cattle-system \
  --set hostname=$YOUR_HOSTNAME \
  --set ingress.tls.source=letsEncrypt \
  --set letsEncrypt.email=$YOUR_EMAIL \
  --set letsEncrypt.ingress.class=nginx \
  --version 2.6.5 \
  --set rancherImageTag=v2.6-head \
  --set extraEnv\[0\].name="CATTLE_PROMETHEUS_METRICS" --set-string extraEnv\[0\].value=true
```

#### Describe the bug

Just a typo with the csp adapter compliance message and support config. Was expecting `license(s)` got `licens(es)`

#### To Reproduce

1. Create an EKS cluster using `eksctl`
2. Install Rancher onto the cluster
3. Install the csp adapter
   1. Setup the required oidc provider & policy
   2. Have premium support license
4. Create a downstream linode cluster with 18 nodes
   - these 18 nodes plus the 3 nodes on the local cluster will take you out of compliance

#### Result
The message in the UI and when downloading the support config has `licens(es)`

#### Expected Result
Expected `license(s)`

#### Screenshots

![2022-06-16_12-47-30](https://user-images.githubusercontent.com/40877149/174178543-7189b745-e3cc-4beb-a2e9-c9332c95ba93.png)

**Comment by MbolotSuse** [comment]
This particular typo should be resolved in version 1.0.0-rc2 of the adapter.

**Comment by brudnak** [comment]
# :white_check_mark: PASSED

## Reproduction Environment

#### Not required.

<hr>
  
## Validation Environment

| Component               | Version / Type                                                                     |
| ----------------------- | ---------------------------------------------------------------------------------- |
| Rancher version         | v2.6-bda0897a42ac77431383eb4a4bcd0454de056b98-head                                 |
| Rancher commit link     | https://github.com/rancher/rancher/commit/bda0897a42ac77431383eb4a4bcd0454de056b98 |
| Installation option     | `eksctl` eks local cluster (bring your own cluster)                                |
| local k8s cluster       | v1.22.10-eks-84b4fe6                                                               |
| Cert Details            | let's encrypt                                                                      |
| Helm version            | v2.16.8-rancher2                                                                   |
| Downstream cluster type | rke1 linode                                                                        |
| Downstream K8s version  | v1.23.6-rancher1-1                                                                 |
| Logged in user role     | administrator                                                                      |
| Browser type            | google chrome                                                                      |
| Browser version         | 102.0.5005.115 (Official Build) (x86_64)                                           |



## Validation steps

1. Create an EKS cluster using `eksctl`
2. Install Rancher onto the cluster
3. Install the csp adapter
   1. Setup the required oidc provider & policy
   2. Have premium support license
4. Create a downstream linode cluster with 18 nodes
   - these 18 nodes plus the 3 nodes on the local cluster will take you out of compliance
5. Observe the compliance banner in the UI. The typo `licens(es)` has been corrected to `license(s)`
  
## Additional Info

## RESULTS

#### :white_check_mark: Expected
Expected license(s)

#### :white_check_mark:Actual
Got license(s)
